pkg/controller/utils/elasticsearch.go

```diff
@@ -310,7 +310,7 @@ func getClientCredentials(client client.Client, ctx context.Context) (string, st
 	}
 
 	esPublicCert := &corev1.Secret{}
-	if err := client.Get(ctx, types.NamespacedName{Name: relasticsearch.PublicCertSecret, Namespace: common.OperatorNamespace()}, esPublicCert); err != nil {
+	if err := client.Get(ctx, types.NamespacedName{Name: relasticsearch.ESGatewayPublicCertSecret, Namespace: common.OperatorNamespace()}, esPublicCert); err != nil {
```

Comment:
It seems incorrect to change this name since here this has nothing to do with the ESGateway.

Reply:
Const name restored to `PublicCertSecret`

Reply:
Oh actually I guess it is related because this is the secret ESGateway presents.... ugh